Library/Formula/postgrest.rb

```diff
@@ -0,0 +1,51 @@
+require "language/haskell"
+require "net/http"
+
+class Postgrest < Formula
+  include Language::Haskell::Cabal
+
+  desc "Serves a fully RESTful API from any existing PostgreSQL database."
+  homepage "https://github.com/begriffs/postgrest"
+  url "https://github.com/begriffs/postgrest/archive/v0.2.12.0.tar.gz"
+  sha256 "9f7277720b947b06eb53ac0a54686eb437253d417695bc756220e703532a725a"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "postgresql"
+
+  setup_ghc_compilers
+
+  def install
+    install_cabal_package
+  end
+
+  test do
+    pg_bin = Formula["postgresql"].bin
+    pg_plist = Formula["postgresql"].opt_prefix/"homebrew.mxcl.postgresql.plist"
+    pg_user = ENV["USER"]
+    test_db = "test_postgrest_formula"
+
+    # Start PostgreSQL server if it is not already running
+    if File.exist? "#{var}/postgres/postmaster.pid"
+      pg_running = true
+    else
+      pg_running = false
+      system "launchctl", "load", "-w", pg_plist
+      sleep(5) # Wait for the server to start
+    end
```

Comment:
I'm not sure I see the disadvantage of piggy-backing on the global version. If I start a dedicated server on a dedicated port, I would have to add logic to find an open port, or risk colliding with something already running on that port (which I'm already doing with the postgrest server).